lib/grandpa/network.go

```diff
@@ -38,9 +38,10 @@ type GrandpaHandshake struct { //nolint:revive
 	Roles byte
 }
 
+// TODO: should we remove the SubProtocol mehtod from the `Message` interface?
```

Comment:
So.. should we? 😄 Also

```suggestion
// TODO: should we remove the SubProtocol method from the `Message` interface?
```